galleries/examples/images_contours_and_fields/image_exact_placement.py

```diff
@@ -0,0 +1,131 @@
+"""
+=========================================
+Placing images, preserving relative sizes
+=========================================
+
+By default Matplotlib resamples images created with `~.Axes.imshow` to
+fit inside the parent `~.Axes`.  This can mean that images that have very
+different original sizes can end up appearing similar in size.
+
+Sometimes, however,  it is desirable to keep the images the same relative size, or
+even to make the images keep exactly the same pixels as the original data.
+Matplotlib does not automatically make either of these things happen,
+but it is possible with some manual manipulation.
+
+Preserving relative sizes
+=========================
+
+If the size of the images are amenable, we can preserve the relative sizes of two
+images by using either the *width_ratio* or *height_ratio* of the subplots.  Which
+one you use depends on the shape of the image and the size of the figure.
+
+By default the two images are made a similar size, despite one being 1.5 times the width
+of the other:
+"""
+
+# sphinx_gallery_thumbnail_number = -1
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import matplotlib.patches as mpatches
+
+# make the data:
+N = 450
+x = np.arange(N) / N
+y = np.arange(N) / N
+
+X, Y = np.meshgrid(x, y)
+R = np.sqrt(X**2 + Y**2)
+f0 = 5
+k = 100
+a = np.sin(np.pi * 2 * (f0 * R + k * R**2 / 2))
+A = a[:100, :300]
+B = A[:40, :200]
+
+# plot with default axes handling:
```

Comment:
```suggestion
# default layout: both axes have the same size
```

Reply:
Ping here: Have you overlooked this? If you don't want to take it, please give a short feedback anyway.